src/redux/news/news-add/news-add.actions.js

```diff
@@ -0,0 +1,8 @@
+import { ADD_NEWS_ITEM } from './news-add.types';
```

Comment:
Please, just have the folder `news` and have files
- `news.types.js`
- `news.actions.js`
- `news.reducer.js`
- `news.sagas.js`

And put all of the actions, types and sagas there. Don't create separate folders inside for each type of operation

Reply:
Also, please, create folder `test` here where you put tests for actions, reducers, and sagas

Reply:
Done!